WPFSKillTree/SkillTreeFiles/Compute.cs

```diff
@@ -624,23 +624,20 @@ public void AttackSpeed(AttackSkill skill, AttributeSet attrs)
                     if (incAS != 0)
                         APS = IncreaseValueByPercentage(APS, incAS);
 
-                    float moreAS = 0;
+                    float moreAS = 1;
                     if (attrs.ContainsKey("#% more Attack Speed"))
-                        moreAS += attrs["#% more Attack Speed"][0];
+                        moreAS *= attrs["#% more Attack Speed"][0] / 100 + 1;
                     if (attrs.ContainsKey("#% less Attack Speed"))
-                        moreAS -= attrs["#% less Attack Speed"][0];
+                        moreAS *= -attrs["#% less Attack Speed"][0] / 100 + 1;
                     foreach (var attr in attrs.Matches(ReMoreAttackSpeedType))
                     {
                         Match m = ReMoreAttackSpeedType.Match(attr.Key);
                         if (m.Success)
-                        {
                             // XXX: Not sure there are any mods with WeaponType here (Melee string in mod is DamageForm now, maybe Unarmed should be form as well).
-                            if (Weapon.Types.ContainsKey(m.Groups[2].Value) && Nature.Is(Weapon.Types[m.Groups[2].Value]))
-                                moreAS += m.Groups[1].Value == "more" ? attr.Value[0] : -attr.Value[0];
-                            else if (DamageNature.Forms.ContainsKey(m.Groups[2].Value) && Nature.Is(DamageNature.Forms[m.Groups[2].Value]))
-                                moreAS += m.Groups[1].Value == "more" ? attr.Value[0] : -attr.Value[0];
-                        }
+                            if (Weapon.Types.ContainsKey(m.Groups[2].Value) && Nature.Is(Weapon.Types[m.Groups[2].Value]) || DamageNature.Forms.ContainsKey(m.Groups[2].Value) && Nature.Is(DamageNature.Forms[m.Groups[2].Value]))
+                                moreAS *= (m.Groups[1].Value == "more" ? attr.Value[0] : -attr.Value[0]) / 100 + 1;
```

Comment:
Same here. There is condition for RegExp subgroup "more" to treat mod as "more AS" otherise as "less AS"